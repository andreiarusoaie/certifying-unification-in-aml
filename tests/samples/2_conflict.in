variables: x
symbols: f, g
problem: g(x) =? f(x)
