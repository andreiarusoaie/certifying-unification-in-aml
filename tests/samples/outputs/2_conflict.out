Found Maude version: 3.0beta1

[1m[95mProof of:[94m (f(x) or g(x) <--> âˆƒ v(3) . v(3) and ((v(3) === f(x)) or (v(3) === g(x))))  [0m
(1)(âˆƒ v(3) . (g(x) <--> v(3) and (v(3) === g(x)))) [e-subst] ;

(2)((g(x) <--> âˆƒ v(3) . v(3) and (v(3) === g(x))) <--> âˆƒ v(3) . (g(x) <--> v(3) and (v(3) === g(x)))) [e-scope] ;

(3)(g(x) <--> âˆƒ v(3) . v(3) and (v(3) === g(x))) [modus-ponens(1, 2)] ;

(4)(âˆƒ v(3) . (f(x) <--> v(3) and (v(3) === f(x)))) [e-subst] ;

(5)((f(x) <--> âˆƒ v(3) . v(3) and (v(3) === f(x))) <--> âˆƒ v(3) . (f(x) <--> v(3) and (v(3) === f(x)))) [e-scope] ;

(6)(f(x) <--> âˆƒ v(3) . v(3) and (v(3) === f(x))) [modus-ponens(4, 5)] ;

(7)(f(x) or g(x) <--> (âˆƒ v(3) . v(3) and (v(3) === f(x))) or (âˆƒ v(3) . v(3) and (v(3) === g(x)))) [eqv-or(3, 6)] ;

(8)((âˆƒ v(3) . v(3) and (v(3) === f(x))) or (âˆƒ v(3) . v(3) and (v(3) === g(x))) <--> âˆƒ v(3) . v(3) and (v(3) === f(x)) or v(3) and (v(3) === g(x))) [e-collapse] ;

(9)(v(3) and ((v(3) === f(x)) or (v(3) === g(x))) <--> v(3) and (v(3) === f(x)) or v(3) and (v(3) === g(x))) [and-or-distr] ;

(10)((âˆƒ v(3) . v(3) and ((v(3) === f(x)) or (v(3) === g(x)))) <--> âˆƒ v(3) . v(3) and (v(3) === f(x)) or v(3) and (v(3) === g(x))) [e-intro(9)] ;

(11)(f(x) or g(x) <--> âˆƒ v(3) . v(3) and (v(3) === f(x)) or v(3) and (v(3) === g(x))) [eqv-tranz(7, 8)] ;

(12)(f(x) or g(x) <--> âˆƒ v(3) . v(3) and ((v(3) === f(x)) or (v(3) === g(x)))) [eqv-tranz(11, 10)] ;
[94mChecked: [0m [1m[92m true
[0m
